osu.Game.Rulesets.Osu/Skinning/LegacyCursor.cs

```diff
@@ -30,13 +32,14 @@ private void load(ISkinSource skin)
                     Anchor = Anchor.Centre,
                     Origin = Anchor.Centre,
                 },
-                new NonPlayfieldSprite
+                cursor = new NonPlayfieldSprite
                 {
                     Texture = skin.GetTexture("cursor"),
                     Anchor = Anchor.Centre,
                     Origin = Anchor.Centre,
                 }
             };
+            cursor.Spin(10000, RotationDirection.Clockwise);
```

Comment:
This duration should probably be compared with stable's - did you check if the rate of spins is the same?